openedx/features/wikimedia_features/admin_dashboard/static/admin_dashboard/js/course_reports.js

```diff
@@ -0,0 +1,80 @@
+(function() {
+    'use strict';
+    var getCookie, ReportDownloads;
+    var url = $("input[name='list-profiles-csv']").data('endpoint') + '/csv';
+    var endpoint =  $('.report-downloads-table').data('endpoint');
+
+    getCookie = function(name) {
+        var cookieValue = null;
+        if (document.cookie && document.cookie !== '') {
+            var cookies = document.cookie.split(';');
+            for (var i = 0; i < cookies.length; i++) {
+                var cookie = jQuery.trim(cookies[i]);
+                // Does this cookie string begin with the name we want?
+                if (cookie.substring(0, name.length + 1) === (name + '=')) {
+                    cookieValue = decodeURIComponent(cookie.substring(name.length + 1));
+                    break;
+                }
+            }
+        }
+        return cookieValue;
+    };
+
+    ReportDownloads = function(){
+        $.ajax({
+            type: 'POST',
+            dataType: 'json',
+            url: endpoint,
+            success: function(data) {
+                if (data.downloads.length){
+                for (let i = 0; i < data.downloads.length; i++) {
+                    $('#report-downloads-table').append(data.downloads[i]['link']);
+                    $('#report-downloads-table').append('<br>');
+                }
+                $('#report-downloads-table').append(data.downloads[0]['link'])
+                }
+            },
+            error: function() {
+                console.log("error in list report downloads api")
```

Comment:
change this message to something like this.`There is an error in the list report downloads API.`